core/mondoo-github-security.mql.yaml

```diff
@@ -106,6 +107,26 @@ queries:
     refs:
       - url: https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-permission-levels-for-an-organization
         title: Setting base permissions for an organization
+  - uid: mondoo-github-organization-security-security-policy
+    title: Ensure repository defines a security policy
+    impact: 30
+    mql: "if ( github.organization.repositories.one(name == \".github\") ) {\n  github.organization.repositories.where( name == \".github\").all( \n    files.one( name.downcase == \"security.md\")\n  ) || github.repository.files.one( name.downcase == \"security.md\")\n} else {\n  github.repository.files.one( name.downcase == \"security.md\")\n}\n"
```

Comment:
this is not properly formatted, use latest version of `cnspec bundle fmt`

Reply:
Fixed